src/lib/index.ts

```diff
@@ -2,6 +2,17 @@ import fetch from 'node-fetch';
 import path from 'path';
 import execSh from 'exec-sh';
 import { specmaticJarPathLocal } from '../config';
+import fs from 'fs';
+
+
+
+export const setSpecmaticEnvVariable = (name: string, value: string, fileName: string) => {
```

Comment:
instead of just (name, value), let's make it ({ name: value}), basically, an object of multiple name values, so that we can update multiple environment variables together

Reply:
Instead of that as a first parameter, you should use the environmentName (In our case we are going to use local) but that should be dynamic

Reply:
Do you mean that we should provide the consumer with ability to update/create a new object inside the environments?
For eg:- if the specmatic.json file only has the baselurl and the user wants to add the variable object to it, so he should be able to add is using this function and also the variable name and its value in a key, value pair right ?

Reply:
no not even baseUrl, the consumer can just set the environment variables